src/actions/guides/frontend/rendering_html.cr

```diff
@@ -64,7 +64,7 @@ class Guides::Frontend::RenderingHtml < GuideAction
 
     You’ll notice we used `needs` near the top of the class. This declares that for
     this page to render we need an Array of Strings and that they will be assigned
-    to the `@user_names` variable. We set the user names by passing it in the
+    to the `user_names` getter method. We set the user names by passing it in the
```

Comment:
I think it might be a little confusing that we show the instance variable, but then say don't use it, or show it being used. If you have a strong understanding of Crystal, then it makes sense, but I think if you're new to both Crystal and Lucky, then you might be a little confused on the instance variable. So not too sure on this.

Reply:
Yeah that is a really good point. Maybe what was confusing me was the "assigned to a getter". Maybe "is accessible from a getter"?

Reply:
Ah, gotcha. That makes sense.